src/cdn_definitions/data.yaml

```diff
@@ -178,7 +178,9 @@ ignore_lp_version_product_ids:
 - "000"
 
 # The RHUI product ID. Used to identify RHUI repos that do not contain the 'rhui' substring.
```

Comment:
There should probably be a sentence or two here explaining the relationship between `rhui_product_id` and `rhui_product_ids` and proposing what clients are supposed to do with them.

For example, clarifying that `rhui_product_ids` takes precedent over `rhui_product_id` (meaning a client should ignore `rhui_product_id` if `rhui_product_ids` is present) - I'm assuming that is the intended behavior.